plumber_rest_api/app/handlers/insert.R

```diff
@@ -0,0 +1,62 @@
+box::use(
+  jsonlite[unserializeJSON, unbox],
+  glue[glue],
+  checkmate[test_string],
+  shiny.telemetry[build_token],
+  stringr[str_sub],
+  logger[log_info, log_debug, log_error, log_warn],
+)
+
+box::use(
+  app/logic/setup[data_storage],
+  app/logic/token[validate_token, get_secret],
+)
+
+#' Handler to insert data to the data storage provider
+#'
+#' @param data seriealizedJSON with an R list object
+#' @param token string with signature of the message
+#' @param id string with identification of which secret to use
+#' @param bucket string that identifies which bucket to save data to
+#' @export
+handler <- function(data, token, id, bucket){
+
+  log_debug('@post /{bucket} triggered')
+
+  values <- tryCatch ({
+    unserializeJSON(data)
+  }, error = function(err) {
+    log_error("Couldn't serialize json")
+    "Couldn't serialize json"
+  })
+
+  if (test_string(values)) {
```

Comment:
Checks if the `unserializeJSON` was successful and fails the request if it's not